src/types.jl

```diff
@@ -359,7 +359,7 @@ struct GlyphCollection
     glyphs::Vector{Char}
     fonts::Vector{FTFont}
     origins::Vector{Point3f}
-    extents::Vector{FreeTypeAbstraction.FontExtent{Float32}}
+    extents::Union{Vector{FreeTypeAbstraction.FontExtent{Float32}}, Vector{TeXChar}}
```

Comment:
Why don't we use the same representation for both? It doesn't have to be FontExtent, but that is just a collection of a couple numbers anyway, so those should be extractable from TeXChar. Basically at the later point in the pipeline, I would like to not differentiate between different ways to generate text anymore.

Reply:
Because I ~am lazy~ wanted to minimize changes for that PR.

That can totally be handled by the the `GlyphCollection` constructor and then stored in an unique way. I will look into it.